### PR TITLE
docs: Add contributing.md file

### PR DESCRIPTION
**Pull request by MaddyUnderStars**
draft contributing guide

I want to include a section on structure, but would like to discuss more with others first about how we will structure these docs.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- [Python Markdown InlineHilite](https://facelessuser.github.io/pymdown-extensions/extensions/inlinehilite/)
+- [Python Markdown Superfences](https://facelessuser.github.io/pymdown-extensions/extensions/superfences/)
+- [Python Markdown Tabbed](https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/)
+
+## Merging to main
+
+Once you're done writing, make a commit containing your changes and push to a new branch or fork.
+Then, make a PR. It will be reviewed, changes requested if necessary, and merged into main.
+
+Do not push directly onto main.
```
Comment (careck) [inline contributing.md:47]: I believe main is actually locked so no one can push to it.

**Review by careck — APPROVED** [review]
Looks good!

File: contributing.md (hunk 1 above)
Comment (MaddyUnderStars) [reply contributing.md:47]: This document actually refers to contributing to the docs itself, which isn't locked at the moment

**Comment by iceaxeliehne** [comment]
This is great Maddy. please merge

**Pull request merged**